app/assets/stylesheets/forms.scss

```diff
@@ -0,0 +1,162 @@
+// Place all the styles related to the form controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
+
+// new page styles
+.whole-container{
+  .header{
+    box-shadow: 0 0 50px 0 aliceblue;
+
+    .container{
+      .pagetitle{
+        padding: 1rem 1rem 0;
+      
+        h2{
+          margin: 0;
+        }
+      }
+    }
+    .nav{
+      display: flex;
+      height: 100%;
+      width: 100%;
+      max-width: 90%;
+      margin: 0 auto;
+
+      .navleft{
+        flex: 1 0 auto;
+        padding: 1rem;
+        display: flex;
+        align-items: center;
+
+        .pagetitle{
+          h2{
+
+          }
+        }
+      }
+      .navright{
+        padding: 1rem;
+        display: flex;
+        align-items: center;
+
+        .pageaction{
+          .btn {
+            background: aliceblue;
+            border: none;
+            padding: 1rem 2rem;
+            border-radius: 8px;
+            cursor: pointer;
+            border: 1px solid #ccc;
+            &:hover{
+              box-shadow: 0 2px 12px 0 burlywood;
+            }
+          }
+        }
+      }
+    }
+
+  }
+  .main-container{
+    padding: 2rem 0;
+    #form-builder{
+      .container{
+        #form-field-panel{
+          .form-field{
+            box-shadow: 0 0 5px .41px darkgray;
+            border-radius: 8px;
+            margin-bottom: 2rem;
+
+            .form-actions{
+              display: flex;
+              padding: 1rem;
+              border-bottom: 2px solid #aaa;
+
+              .left{
+                flex: 1 0 auto;
+              }
+              .right{}
+              .form-options{
+                div{
+                  .action-btn{
+                    background: #fff;
+                    border-radius: 4px;
+                    font-weight: bolder;
+                    padding: 6px 12px;
+                    cursor: pointer;
+                  }
+                }
+              }
+              .main{
+                div{
+                  .action-btn{
+                    border: 1px solid rgb(5, 80, 241);
```

Comment:
you can create color as scss variables so that you can avoid making multiple duplicates